internal/status/dashboard.go

```diff
@@ -174,6 +174,7 @@ func (d *dashboard) handleIndex(w http.ResponseWriter, r *http.Request) {
 		Tool:     d.spec.Tool,
 		Statuses: statuses,
 	}
+	// TODO: unhandled error, would it be reasonable to panic here?
 	indexTemplate.Execute(w, content)
```

Comment:
I believe the only way for this `template.Execute` to error would be a programming error, the data being passed to the template does not match the values expected by the template, or the template itself has a syntax error.

Would you be ok with a panic here? I believe this category of error would generally be caught during development.

Another option would be to log the error, but I'm not sure how to get the appropriate logger in this HTTP handler.

Reply:
Panic seems good to me!